Include <unistd.h>.

Fixes:

error: use of undeclared identifier 'close'

Change 6 of 6 from
https://sourceforge.net/p/gelide/code/ci/b8da37d825b9548ba6548f96ce8efb84cf068001/
--- src/utils/process.cpp
+++ src/utils/process.cpp
@@ -23,7 +23,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <glibmm.h>
-
+#include <unistd.h>
 
 CProcess::CProcess(void): m_proc_id(-1){
 }
